src/test/rustdoc-ui/invalid-html-tags.rs

```diff
@@ -73,3 +73,13 @@ pub fn e() {}
 /// <div></div
 //~^ ERROR unclosed HTML tag `div`
 pub fn f() {}
+
+/// <!---->
+/// <!-- -->
+/// <!-- <div> -->
+/// <!-- <!-- -->
```

Comment:
How about
```suggestion
/// <!-- <!-- -->
/// -->
/// <!--
```
Two things, one is opening without closing, and vice versa.

Reply:
No, the test is actually to check that `<!--` inside a comment is ignored as it should.